### DISCUSSION TITLE
docs/linux/gaming/apex-legends/

### DISCUSSION THREAD

**Discussion by giscus** [userliluzibert/userliluzibert.github.io#5] [Giscus]
# docs/linux/gaming/apex-legends/

Image stolen from https://store.steampowered.com/app/1172470/Apex_Legends/
So over my winter break I’ve been doing a little gaming here and there. One day, my friends invited me to play Apex Legends. So I decided to hop check out protondb and give it a try. Much to my luck Apex Legends actually works (with a few tweaks).
Pretty much, all you have to do is run this line through the Steam launcher’s launch options:

https://liluzibird.github.io/docs/linux/gaming/apex-legends/

**Comment by LuisRuv21** [comment]
We didn't win a single game that day, LOL (Other than the team based modes where other players carried)

**Reply by liluzibird** [reply]
Bro, the recoil is so weird. I've been playing CS for too long. Still bad at that game also 🤣

**Reply by LuisRuv21** [reply]
I've been playing the game for years and taking the semester off from playing made me garbage, LOL

**Reply by liluzibird** [reply]
We should drop out and become eSports players 🎮🏅
